task2-bash/guess.sh

```diff
@@ -34,7 +34,7 @@ run_game() {
 		((total_tries++))
 	else
 		echo "Something goes wrong..."
-		exit 2;
+		exit 10;
```

Comment:
It's a good practice to define constants for the exit codes for scripts non-oneliner scripts

Reply:
Fixed https://github.com/Kernel-GL-HRK/gl_kernel_BaseCamp_2022_II/pull/33/commits/265aead3cd1e5b3a4d26ad359b734fcddd1a1d6f